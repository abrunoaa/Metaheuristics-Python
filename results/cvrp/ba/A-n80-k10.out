elapsed;gap;fitness;solution
19.17s;3.85706%;1831;[[1, 7, 21, 40], [10, 71, 28, 79, 18, 48, 14], [12, 44, 5, 23, 11, 52, 63], [13, 29, 27, 59, 6, 24, 62], [20, 75, 57, 19, 47, 26, 35, 65, 69, 56, 15, 64, 36], [30, 78, 61, 16, 43, 68, 8, 37, 2, 34], [42, 51, 53, 67, 66, 70, 38, 49], [54, 9, 55, 33, 41, 25, 46, 74], [58, 32, 4, 22, 45, 50, 72, 76], [73, 39, 60, 31, 17, 3, 77]]
13.29s;2.60919%;1809;[[1, 7, 21, 40], [3, 77, 51, 42, 73, 49], [5, 30, 6, 24, 8, 37, 2, 34], [10, 71, 63, 62, 23, 12], [11, 52, 28, 79, 18, 48, 14], [13, 74, 17, 31, 60, 39, 53], [27, 59, 19, 26, 35, 65, 69, 56, 47, 41, 25, 46], [29, 20, 75, 57, 61, 16, 43, 68, 78, 44], [36, 66, 70, 76, 50, 54, 9, 55, 15, 33, 64], [38, 58, 32, 4, 22, 45, 72, 67]]
2.17s;4.59444%;1844;[[1, 7, 21, 40], [3, 39, 60, 31, 17, 74], [10, 71, 28, 79, 18, 48, 14], [12, 5, 59, 20, 75, 27, 29, 13], [23, 24, 6, 8, 68, 43, 16, 61, 78], [42, 36, 66, 67, 38, 49], [44, 30, 57, 26, 35, 65, 69, 56, 47, 19, 25, 46], [51, 77, 64, 41, 15, 33, 55, 9, 54, 72, 76, 70, 53], [58, 32, 4, 22, 45, 50, 73], [62, 37, 2, 34, 52, 11, 63]]
15.97s;3.1764%;1819;[[10, 71, 28, 79, 18, 48, 14], [12, 44, 23, 11, 52, 63, 62], [13, 1, 7, 21, 40], [17, 31, 27, 59, 25, 41, 46, 60], [24, 8, 68, 43, 16, 61, 57, 75, 20, 78], [29, 5, 30, 6, 37, 2, 34], [32, 4, 22, 45, 50, 72, 76, 70], [36, 66, 67, 58, 38, 73, 49], [42, 51, 77, 3, 39, 74], [53, 64, 33, 15, 47, 19, 26, 35, 65, 69, 56, 55, 9, 54]]
6.79s;2.43902%;1806;[[1, 7, 21, 40], [2, 37, 8, 68, 43, 16, 61, 78, 13], [3, 39, 60, 29, 74], [10, 71, 28, 79, 18, 48, 14], [17, 31, 27, 59, 5, 44, 12, 62], [20, 75, 57, 25, 41, 46, 64, 77, 51], [23, 30, 6, 24, 34, 52, 11, 63], [33, 15, 47, 19, 26, 35, 65, 69, 56, 55, 9, 54, 72, 50, 58], [38, 32, 4, 22, 45, 76], [42, 53, 70, 66, 67, 36, 73, 49]]
9.22s;3.74362%;1829;[[1, 7, 21, 40], [3, 17, 31, 59, 5, 44, 12], [10, 71, 28, 79, 18, 48, 14], [13, 74, 29, 60, 39, 53], [23, 30, 78, 61, 75, 19, 47, 26, 35, 65, 69, 56, 64], [24, 6, 8, 68, 16, 43, 57, 20, 27], [42, 36, 66, 67, 70, 38, 73], [49, 58, 32, 4, 22, 45, 50], [51, 77, 46, 25, 41, 15, 33, 55, 9, 54, 72, 76], [52, 2, 37, 34, 11, 63, 62]]
4.33s;3.9705%;1833;[[1, 12, 44, 5, 59, 27, 31, 29], [11, 34, 2, 37, 6, 24, 23], [13, 74, 60, 39, 3, 77], [17, 19, 47, 56, 69, 65, 35, 26, 57, 75, 20], [21, 14, 48, 18, 79, 28, 52], [30, 78, 8, 68, 43, 16, 61, 25, 41, 46, 51], [38, 76, 72, 54, 9, 55, 15, 33, 64, 53], [40, 7, 10, 71, 63, 62], [42, 67, 66, 36, 73], [49, 58, 32, 4, 22, 45, 50, 70]]
8.42s;3.51673%;1825;[[1, 7, 40], [3, 31, 57, 61, 16, 43, 68, 78, 30], [10, 71, 28, 79, 18, 48, 14], [12, 44, 5, 62, 11, 63, 21], [13, 74, 17, 60, 39, 51], [23, 24, 6, 8, 37, 2, 34, 52], [29, 27, 59, 20, 75, 25, 41, 46, 77], [42, 36, 66, 67, 38, 73, 49], [53, 70, 72, 45, 22, 4, 32, 50, 58], [64, 33, 15, 47, 19, 26, 35, 65, 69, 56, 55, 9, 54, 76]]
5.80s;3.63018%;1827;[[1, 7, 21, 40], [10, 71, 34, 2, 37, 8, 6, 24], [11, 52, 28, 79, 18, 48, 14], [12, 44, 5, 59, 27, 17, 74], [13, 39, 29, 23, 62, 63], [30, 78, 68, 43, 16, 61, 57, 75, 20, 31], [36, 67, 66, 70, 38, 73, 49], [42, 51, 77, 3, 64, 33, 15, 55, 9, 54], [53, 41, 47, 56, 69, 65, 35, 26, 19, 25, 46, 60], [58, 32, 4, 22, 45, 50, 72, 76]]
6.62s;5.04821%;1852;[[1, 7, 71, 10, 21, 40], [3, 60, 39, 53, 73], [11, 52, 28, 79, 18, 48, 14], [13, 29, 5, 59, 27, 31, 17, 74], [20, 75, 57, 61, 16, 43, 68, 78, 30, 44], [24, 6, 8, 37, 2, 34, 63], [36, 66, 67, 76, 72, 54, 9, 50, 49], [38, 58, 32, 4, 22, 45, 70], [42, 51, 12, 23, 62], [64, 33, 15, 55, 56, 69, 65, 35, 26, 19, 47, 41, 25, 46, 77]]

avg_time;avg_ans;best_ans;worst_ans;avg_gap;best_gap;worst_gap
9.18s;1827.50;1806;1852;3.65854%;2.43902%;5.04821%
