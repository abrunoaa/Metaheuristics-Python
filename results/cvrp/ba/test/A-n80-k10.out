elapsed;gap;fitness;solution
15.91s;2.94952%;1815;[[1, 7, 21, 40], [5, 78, 68, 43, 16, 61, 57, 75, 20, 31], [10, 11, 24, 6, 23, 62, 13], [12, 44, 30, 8, 37, 2, 34, 63], [27, 59, 19, 26, 35, 65, 69, 56, 47, 41, 25, 46], [36, 70, 50, 45, 22, 4, 32, 58], [38, 76, 72, 54, 9, 55, 15, 33, 64, 51], [42, 3, 77, 53, 73, 49], [52, 28, 79, 18, 48, 14, 71], [67, 66, 39, 60, 17, 29, 74]]
7.65s;2.94952%;1815;[[1, 7, 40], [3, 74, 17, 60, 39, 77], [5, 30, 78, 68, 43, 16, 61, 59, 27, 29], [11, 52, 28, 79, 18, 48, 14], [12, 44, 62, 63, 71, 10, 21], [13, 51, 31, 20, 75, 57, 19, 47, 26, 35, 65, 69, 56, 15, 33], [23, 24, 6, 8, 37, 2, 34], [42, 36, 66, 67, 70, 58, 38, 49], [53, 64, 46, 25, 41, 55, 9, 54, 72], [73, 32, 4, 22, 45, 50, 76]]
10.27s;3.46001%;1824;[[1, 7, 21, 40], [10, 71, 28, 79, 18, 48, 14], [12, 44, 5, 30, 6, 23, 62], [13, 74, 17, 3, 42], [24, 8, 37, 2, 34, 52, 11, 63], [27, 59, 20, 75, 25, 41, 46, 60, 77], [29, 31, 19, 57, 61, 16, 43, 68, 78], [36, 53, 66, 70, 38, 73, 49], [51, 39, 64, 33, 15, 47, 26, 35, 65, 69, 56, 55, 9, 54, 72], [58, 32, 4, 22, 45, 50, 76, 67]]

avg_time;avg_ans;best_ans;worst_ans;avg_gap;best_gap;worst_gap
11.28s;1818.00;1815;1824;3.11968%;2.94952%;3.46001%
